src/services/TraineeProfileService.ts

```diff
@@ -0,0 +1,16 @@
+import { AxiosResponse } from "axios";
+import ApiService from "./apiService";
+import { TraineeProfile } from "../models/TraineeProfile";
+
+export class TraineeProfileService {
+  private apiService: ApiService;
+  constructor() {
+    super("/trainee/api");
```

Comment:
This TraineeProfileService doesn't have a parent class, but calls super() in constructor. And SonarCloud reports a bug here as well.

Reply:
Updated.